deploy/docker/env2js.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+#
+# Generate a Javascript config file from environment variables.
+# Used by Docker startup script to inject run-time config to Kiali React app
+
+if [ ! -z ${SERVER_WEB_ROOT} ]; then
+cat <<END
+window.WEB_ROOT='${SERVER_WEB_ROOT}';
+END
+fi
```

Comment:
Why is this a separate script? Looks like its only called from kiali-server.sh in one place - just put it right in kiali-server.sh, thus no need to have to manage a completely separate script.